src/SelectField/SelectField.js

```diff
@@ -180,6 +185,7 @@ class SelectField extends Component {
       <TextField
         {...other}
         style={style}
+        defaultText={hintText || floatingLabelText ? '' : defaultText}
```

Comment:
> we'll use the first option's value?

I prefer this alternative as we would have solve it the same way with a regular `<select />` right?